inventory/group_vars/k8s-cluster.yml

```diff
@@ -138,3 +138,7 @@ helm_enabled: false
 # dnsmasq_upstream_dns_servers:
 #  - /resolvethiszone.with/10.0.4.250
 #  - 8.8.8.8
+
+# be sure of what you do when updating following variables
```

Comment:
I'd like to see something a bit more descriptive in the vars file on what these settings do, where I can (as a new user) go to learn more, etc.. Other than that, looks good!